docs/deployment.md

```diff
@@ -0,0 +1,89 @@
+# ProjectBuilder Deployment
+
+There are mainly three different deployments via docker-compose scripts:
+* development deployment
+* remote deployment
+* deployment on kubernetes cluster
+
+The following picture depicts the deployed softwarepart and the interaction with build & deployment automation.
+
+![Overall picture](diagrams/deployment-current.drawio.svg)
+
+## Development Deployment (private, with docker-compose)
+
+For development, ProjectBuilder is running native with java command in order to enable debugging and fast changes. in deployment folder are two docker-compose scripts:
+* localenv-docker-compose.yml: combines keycloak and mariaDB
+* mysqllocal-docker-compose.yml: deploys only mariaDB, if it is not needed to run ProjectBuilder with Keycloak (dev-mod)
+
+## Server Deployment (public, with docker-compose)
+
+:heavy_exclamation_mark: This deployment is not recommended for high user-counts. A downtime is needed for doing updates. Use e.g. Kubernetes deployment to ensure a failsave and stable environment.
+
+### Usage of different scripts
+
+The scripts for server deployment are located under `deployment\https`. The following software is deployed:
+
+* certbot + nginx: provides ssl-certificate-handling in oder to do https
+* keycloak: authentication and authorisation. For development, keycloak is using integrated database. In production an additional MariaDB is used
+* mariaDB: Database for projectBuilder
+* mariaDB for keycloak
+
+In order to provided different release cicles for the different software products, three compose scripts are used:
+
+* docker-compose.yml: used during the installation of from init-letsencrypt.sh in order to integrate certbot ssl-secret-handling into nginx
+* env-docker-compose.yml: includes nginx, keycloak, which are only updated when new versions are provided by producers
+* projectbuildr-docker-compose.yml: includes projectBuilder with database and is updated on new projectBuilder release - which should be very often
+
+### Installation Steps
+
+#### Initial Installation
+
+1) Install prerequisites. See [documentation for installation of prerequisites](installation/prerequisites-installation.md) for details.
+2) checkout projectbuilder deployment folder
+3) go to https folder
+4) set variables in env.sh and add it as source:
```

Comment:
```suggestion
1) Install prerequisites. See [documentation for installation of prerequisites](installation/prerequisites-installation.md) for details.
2) checkout projectbuilder deployment folder
3) go to `https` folder
4) set variables in env.sh and add it as a source:
```

Reply:
Did you mean, that you should source the variables from the env file? Then 4) should be restructured